include/sir/config.h

```diff
@@ -93,12 +93,23 @@
 # define SIR_PIDSUFFIX ")"
 
 /**
- * The format for the current process/thread ID.
+ * The format for the current process ID.
  *
- * @remark Only applies if ::SIRO_NOPID or ::SIRO_NOTID are not set.
+ * @remark Only applies if ::SIRO_NOPID is not set.
  */
 # define SIR_PIDFORMAT "%d"
 
+/**
+ * The format for the current thread ID.
+ *
+ * @remark Only applies if ::SIRO_NOTID is not set.
+ */
+# if defined(__NetBSD__)
```

Comment:
why don't we change this to `__USE_HEX_TIDS__`, and define it in platform so we don't have to put all the platforms in this header? thoughts?

thinking ahead to when we have 4-5 here

Reply:
Sure ...